cape-ph/cape-cod
# 142 move dap api behind alb and route53

# TO TEST

This has been deployed and tested to make sure API endpoints are available via custom domain name within the VPN. A custom build of the UI has also been put out there with the new custom domains for the api.

Best way to test is to get on the VPN and use the UI to make sure it has only 200's for our custom domains and to also hit the api endpoint with curl. format:

`curl -k https://api.cape-dev.org/dap-dev/analysispipelines`
